src/signup/signup.js

```diff
@@ -75,6 +117,16 @@ class SignupComponent extends React.Component {
               Submit
             </Button>
           </form>
+          {this.state.signupError ? (
```

Comment:
Quick tip: you're doing an if/else statement here, but inside the else you have null. 
There's a better way to do this;

```
{this.state.signupError && (
  <Typography></Typography>
)}
```

You can learn about it here https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
So good to know, thank you! I read the documentation and changed the if/else statement.